extensions/avro-extensions/src/test/java/io/druid/data/input/AvroHadoopInputRowParserTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.druid.data.input;
+
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.mapred.AvroValue;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.druid.data.input.AvroStreamInputRowParserTest.PARSE_SPEC;
+import static io.druid.data.input.AvroStreamInputRowParserTest.assertInputRowCorrect;
+import static io.druid.data.input.AvroStreamInputRowParserTest.buildSomeAvroDatum;
+
+public class AvroHadoopInputRowParserTest
+{
+  private AvroHadoopInputRowParser parser;
+
+  @Before
+  public void before()
+  {
+    parser = new AvroHadoopInputRowParser(PARSE_SPEC);
+  }
+
+  @Test
+  public void testParse()
+  {
+    InputRow inputRow = parser.parse(new AvroValue<GenericRecord>(buildSomeAvroDatum()));
+    assertInputRowCorrect(inputRow);
+  }
+}
```

Comment:
pls see https://github.com/druid-io/druid-api/blob/master/src/test/java/io/druid/data/input/impl/InputRowParserSerdeTest.java#L42 for example.